locales/ta_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -59,121 +60,117 @@ msgstr "%s மணி நேரங்களுக்கு முன்னர்"
 
 msgctxt "showcase_donations"
 msgid "%s in privacy donations!"
-msgstr "தனியுரிமைக்கான நன்கொடையில் %s"
+msgstr "தனியுரிமைக்கான நன்கொடையில் %s !"
 
 #. Places distance in km
 msgid "%s km"
 msgstr "%s கிமீ"
 
 #. Places distance in miles
 msgid "%s mi"
-msgstr ""
+msgstr "%s மைல்"
 
 #. This is used to add attribution to user reviews on websites, e.g. "John Doe on Yelp"
 msgctxt "maps_places"
 msgid "%s on %s"
-msgstr ""
+msgstr "%s அங்கே%s"
 
 msgctxt "install-duckduckgo"
 msgid "%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
-msgstr ""
+msgstr "%s %sசேர்%sஐத் தேடு %sஅனுமதி%s, என்பதைக் கிளிக் செய்து %sசரி%s"
```

Comment:
The placeholders here seem misplaced. Again, I don't really understand your language and I'm just spot checking with machine translation, so I may be mistaken. The placeholders here are markup elements. If you're familiar with html, this may help, otherwise please ignore, but the placeholders stand for `"<li>","<b>","</b></li><li>","<b>","</b>", "<b>","</b></li>"`.

Sorry if this is confusing, feel free to comment further and I'll be happy to help.

Can you also check the next few ones?

Reply:
The convention in our language is like: I hate Google -----> I Google hate
It will be helpful if you can show me the format for html that I have
to follow to translate the following.
I will provide the translation format.
1) add click allow check , then okay click.
2) allow click add click allow check, then Okay click.
3) continue to installation click add click allow check, then Okay click.

Reply:
Unfortunately these strings are poorly set up. We should have created more discreet strings and most importantly we shouldn't have lumped all these markup elements together. I'd appreciate it if you could find a way to work around this.

This string 

```
%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s
```

becomes

```html
<li>Click <b>Add</b></li><li>Check <b>Allow</b>, then click <b>Okay</b></li>
```

Placeholders are substituted one by one with these `"<li>","<b>","</b></li><li>","<b>","</b>", "<b>","</b></li>"`.

Something similar happens with the other strings.

Does this help? Is it possible to maintain formatting for these strings?

Reply:
I think I got a rough idea of this. Let me translate and show you, You can check.

Reply:
Please check if this translation is the right one:
<li><b>சேர்</b> ஐ கிளிக் செய்யவும்</li><li><b>அனுமதி</b> ஐ சரிபார்து, பின்னர் <b>சரி</b> ஐ கிளிக் செய்யவும்</li>

Reply:
`<li><b>சேர்</b> ஐ கிளிக் செய்யவும்</li><li><b>அனுமதி</b> ஐ சரிபார்து, பின்னர் <b>சரி</b> ஐ கிளிக் செய்யவும்</li>`

Reply:
I'm afraid there's no way to obtain that string with the existing placeholders. Specifically, if you look at the list `"<li>","<b>","</b></li><li>","<b>","</b>", "<b>","</b></li>"`, there's no option for a standalone `</li>` and there is a single standalone `</b>`, while you're using 3. Again, sorry if this is confusing and if the string is genuinely hard (or impossible) to translate appropriately.

Reply:
Since we are having trouble in translating these lines, I will leave these 3-4 lines as it is and mark as needs work(I'm using POedit).
Someone in future or me if i could solve this problem will make sure to update.
As this is basic english I am more than sure that Tamil people can understand this, but I will make sure to solve this problem asap.
Till then you can update the rest of the translations.

Reply:
@superjoe12 It's probably better if you leave those strings empty, rather than in English. This way they will show up as untranslated in Poedit (and similar tools).

Reply:
Ok sure.